library/std/src/primitive_docs.rs

```diff
@@ -100,7 +100,7 @@ mod prim_bool {}
 /// behaviour of the `!` type - expressions with type `!` will coerce into any other type.
 ///
 /// [`u32`]: prim@u32
-/// [`exit`]: process::exit
+#[doc = concat!("[`exit`]: ", include_str!(concat!("../../", env!("CARGO_PKG_NAME"), "/primitive_docs/process_exit.md")))]
```

Comment:
I tried a macro to make this simpler, but it doesn't let me emit single attributes without an item:

```
error: expected item after attributes
   --> library/core/src/../../std/src/primitive_docs.rs:3:3
    |
1   | / macro_rules! std_link {
2   | |     ($name: literal, $file: literal) => {
3   | |         #[doc = concat!("[", $name, "]: ", include_str!(concat!("../../", env!("CARGO_PKG_NAME"), "/primitive_docs/", $file, ".md")))]
    | |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
4   | |     }
5   | | }
    | |_- in this expansion of `std_link!`
...
109 |   std_link!("`exit`", "process_exit");
    |   ------------------------------------ in this macro invocation
```
and wrapping the whole thing in a macro seems like more trouble than it's worth.

Reply:
It's fine as is.